### PR TITLE
[ENT-529] rough draft of some changes to the SSO setup for the auth o…

### PR DESCRIPTION
**Pull request by aaronbini**
## What are you changing in this pull request and why?

Rough draft of some changes to SSO setup for the auth overhaul
https://dbtlabs.atlassian.net/browse/ENTERPRISE-529

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/deploy/regions-ip-addresses.md [hunk 1]
```diff
@@ -9,12 +9,12 @@ dbt Cloud is [hosted](/docs/deploy/architecture) in multiple regions and will al
 [dbt Cloud Enterprise](https://www.getdbt.com/pricing/) plans can choose to have their account hosted in any of the below regions. Organizations **must** choose a single region per dbt Cloud account. If you need to run dbt Cloud in multiple regions, we recommend using multiple dbt Cloud accounts. 
 
 
-| Region | Location | Access URL | IP addresses | Developer plan | Team plan | Enterprise plan |
-|--------|----------|------------|--------------|-----------------|------------|------------------|
-| North America [^1] | AWS us-east-1 (N. Virginia) | cloud.getdbt.com | 52.45.144.63 <br /> 54.81.134.249 <br />52.22.161.231 | ✅ | ✅ | ✅ |
-| EMEA [^1] | AWS eu-central-1	(Frankfurt) | emea.dbt.com | 3.123.45.39 <br /> 3.126.140.248 <br /> 3.72.153.148 | ❌ | ❌ | ✅ |
-| APAC  [^1] | 	AWS ap-southeast-2  (Sydney)| au.dbt.com | 52.65.89.235 <br /> 3.106.40.33 <br /> 13.239.155.206 <br />| ❌ | ❌ | ✅ |
-| Virtual Private dbt or Single tenant | Customized |  Customized | Ask [Support](/guides/legacy/getting-help#dbt-cloud-support) for your IPs | ❌ | ❌ | ✅ |
+| Region | Location | Access URL | Auth URL | Audience URN | IP addresses | Developer plan | Team plan | Enterprise plan |
```
Comment (aaronbini) [inline website/docs/docs/deploy/regions-ip-addresses.md]: these changes may be better suited for different table in a different page, but I put them here in this rough draft

File: website/docs/docs/deploy/regions-ip-addresses.md [hunk 2]
```diff
@@ -9,12 +9,12 @@ dbt Cloud is [hosted](/docs/deploy/architecture) in multiple regions and will al
...
+|--------|----------|------------|------------|------------|--------------|-----------------|------------|------------------|
+| North America [^1] | AWS us-east-1 (N. Virginia) | cloud.getdbt.com | us-production-mt.us.auth0.com | urn:auth0:us-production-mt | 52.45.144.63 <br /> 54.81.134.249 <br />52.22.161.231 | ✅ | ✅ | ✅ |
```
Comment (ajaythomas) [inline website/docs/docs/deploy/regions-ip-addresses.md]: @robotmiller after this [ticket](https://dbtlabs.atlassian.net/browse/ENTERPRISE-356), auth url will be a dbt.com domain-ed one right?

**Review by ajaythomas — COMMENTED** [review]
Where do I see the new docs? I can preview your changes here https://deploy-preview-2933--docs-getdbt-com.netlify.app/docs/collaborate/manage-access/set-up-sso-okta but where do i go to `/next`

File: website/docs/docs/deploy/regions-ip-addresses.md (hunk 2 above)
Comment (robotmiller) [reply website/docs/docs/deploy/regions-ip-addresses.md]: Yes, that's correct!

**Comment by aaronbini** [comment]
> Where do I see the new docs? I can preview your changes here https://deploy-preview-2933--docs-getdbt-com.netlify.app/docs/collaborate/manage-access/set-up-sso-okta but where do i go to `/next`

you can preview the changes at that preview url, and then since this PR targets the `/next` branch, once merged it would show up at `https://next.docs.getdbt.com/docs/`

File: website/docs/docs/cloud/about-cloud/regions-ip-addresses.md [hunk 3]
```diff
@@ -9,12 +9,12 @@ dbt Cloud is [hosted](/docs/cloud/about-cloud/architecture) in multiple regions
 [dbt Cloud Enterprise](https://www.getdbt.com/pricing/) plans can choose to have their account hosted in any of the below regions. Organizations **must** choose a single region per dbt Cloud account. If you need to run dbt Cloud in multiple regions, we recommend using multiple dbt Cloud accounts. 
 
 
-| Region | Location | Access URL | IP addresses | Developer plan | Team plan | Enterprise plan |
-|--------|----------|------------|--------------|-----------------|------------|------------------|
-| North America [^1] | AWS us-east-1 (N. Virginia) | cloud.getdbt.com | 52.45.144.63 <br /> 54.81.134.249 <br />52.22.161.231 | ✅ | ✅ | ✅ |
-| EMEA [^1] | AWS eu-central-1	(Frankfurt) | emea.dbt.com | 3.123.45.39 <br /> 3.126.140.248 <br /> 3.72.153.148 | ❌ | ❌ | ✅ |
-| APAC  [^1] | 	AWS ap-southeast-2  (Sydney)| au.dbt.com | 52.65.89.235 <br /> 3.106.40.33 <br /> 13.239.155.206 <br />| ❌ | ❌ | ✅ |
-| Virtual Private dbt or Single tenant | Customized |  Customized | Ask [Support](/guides/legacy/getting-help#dbt-cloud-support) for your IPs | ❌ | ❌ | ✅ |
+| Region | Location | Access URL | Auth URL | Audience URN | IP addresses | Developer plan | Team plan | Enterprise plan |
```
Comment (aaronbini) [inline website/docs/docs/cloud/about-cloud/regions-ip-addresses.md:12]: these changes may be better suited for a new table in a different page, but I put them here in this rough draft

Comment (matthewshaver) [reply website/docs/docs/cloud/about-cloud/regions-ip-addresses.md:12]: Note to myself to turn this into a snippet down the road.

**Review by robotmiller — DISMISSED** [review]

File: website/docs/docs/cloud/manage-access/set-up-sso-google-workspace.md [hunk 4]
```diff
@@ -57,14 +57,14 @@ Client Secret for use in dbt Cloud.
...
-7. Use the following configuration values when creating your Credentials, replacing `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/cloud/about-cloud/regions-ip-addresses) for your region and plan.
+7. Use the following configuration values when creating your Credentials, replacing `YOUR_ACCESS_URL` and `YOUR_AUTH_URL` with the [appropriate Access URL and Auth URL](/docs/cloud/about-cloud/regions-ip-addresses) for your region and plan.
 
 | Config | Value |
 | ------ | ----- |
 | **Application type** | Web application |
 | **Name** | dbt Cloud |
 | **Authorized Javascript origins** | `https://YOUR_ACCESS_URL` |
-| **Authorized Redirect URIs** | `https://YOUR_ACCESS_URL/complete/gsuite` |
+| **Authorized Redirect URIs** | `https://YOUR_AUTH_URL/login/callback?connection=<login slug>` |
```
Comment (robotmiller) [inline website/docs/docs/cloud/manage-access/set-up-sso-google-workspace.md]: This should just be `https://YOUR_AUTH_URL/login/callback`, the connection parameter isn't needed.

File: website/docs/docs/cloud/manage-access/set-up-sso-azure-active-directory.md [hunk 5]
```diff
@@ -41,13 +41,13 @@ need to select the appropriate directory and then register a new application.
...
-   enterprise use-cases, you will want to use the single-tenant Redirect URI. Replace `YOUR_ACCESS_URL` with the [appropriate Access URL](/docs/cloud/about-cloud/regions-ip-addresses) for your region and plan.
+   enterprise use-cases, you will want to use the single-tenant Redirect URI. Replace `YOUR_AUTH_URL` with the [appropriate Auth URL](/docs/cloud/about-cloud/regions-ip-addresses) for your region and plan.
 
 
 | Application Type | Redirect URI |
 | ----- | ----- |
-| Single-Tenant _(recommended)_ | `https://YOUR_ACCESS_URL/complete/azure_single_tenant` |
-| Multi-Tenant | `https://YOUR_ACCESS_URL/complete/azure_multi_tenant` |
+| Single-Tenant _(recommended)_ | `https://YOUR_AUTH_URL/login/callback?connection=<login slug>` |
+| Multi-Tenant | `https://YOUR_AUTH_URL/login/callback?connection=<login slug>` |
```
Comment (robotmiller) [inline website/docs/docs/cloud/manage-access/set-up-sso-azure-active-directory.md]: I'm pretty sure these both don't need the `connection` parameter either, but will confirm.

File: website/docs/docs/cloud/manage-access/set-up-sso-google-workspace.md (hunk 4 above)
Comment (matthewshaver) [inline website/docs/docs/cloud/manage-access/set-up-sso-google-workspace.md]:
```suggestion
| **Authorized Redirect URIs** | `https://YOUR_AUTH_URL/login/callback` |
```

File: website/docs/docs/cloud/manage-access/set-up-sso-azure-active-directory.md (hunk 5 above)
Comment (matthewshaver) [inline website/docs/docs/cloud/manage-access/set-up-sso-azure-active-directory.md]:
```suggestion
| Multi-Tenant | `https://YOUR_AUTH_URL/login/callback` |
```

**Review by matthewshaver — APPROVED** [review]

**Pull request merged**